libs/sdk-backend-tiger/src/backend/workspace/dateFilterConfigs/index.ts

```diff
@@ -24,8 +24,6 @@ const DefaultDateFilterConfig: IDateFilterConfig = {
     },
     relativeForm: {
         type: "relativeForm",
-        // month has to be the first as it should be the default selected option
-        availableGranularities: ["GDC.time.month", "GDC.time.date", "GDC.time.quarter", "GDC.time.year"],
```

Comment:
So what will this config do now? Will there still be the granularities available? Or do we need to move this to the "global" property of this type?